Client/Frontend/Toolbar+URLBar/TabLocationView.swift

```diff
@@ -416,7 +416,7 @@ extension TabLocationView: TabEventHandler {
             trackingProtectionButton.alpha = 1.0
             let themeManager: ThemeManager = AppContainer.shared.resolve()
             self.blockerStatus = blocker.status
-            self.hasSecureContent = !(tab.webView?.hasOnlySecureContent ?? false)
```

Comment:
@thatswinnie 😰 I assumed that `tab.webView.hasOnlySecureContent` returns `true` if the website is secure.

Reply:
@lmarceau could you take a look at this? I think you touched the code last and probably know this area better.

Reply:
The change looks fine to me!